test/DocumentFormat.OpenXml.Tests/ofapiTest/OpenXmlReaderTest.cs

```diff
@@ -385,6 +385,41 @@ public void PartReaderMiscNodeTest()
             reader.Close();
         }
 
+        /// <summary>
+        ///A test for OpenXmlPartReader to test the ignoreWhitespace option
+        ///</summary>
+        [Theory]
+        [InlineData(true)]
+        [InlineData(false)]
+        public void PartReaderIgnoreWhitespaceTest(bool ignoreWhitespace)
+        {
+            string partText = "<w:document xmlns:w=\"http://schemas.openxmlformats.org/wordprocessingml/2006/main\">" +
+                                "<w:body>" +
+                                    "<w:p w:rsidP=\"001\"><w:r><w:t>  </w:t></w:r></w:p>" +
+                               "</w:body>" +
+                              "</w:document>";
+
+            UTF8Encoding utf8Encoding = new UTF8Encoding();
+            Stream stream = new MemoryStream(utf8Encoding.GetBytes(partText), false);
+
+            //======== new test with a new reader ========
+            OpenXmlReader reader = OpenXmlReader.Create(stream, false, ignoreWhitespace); // ignore whitespaces
+            Assert.False(reader.EOF);
+
+            reader.Read();
+            Assert.False(reader.EOF);
+
+            reader.ReadFirstChild(); // to <w:body>
```

Comment:
Assert each of these instead of commenting them

Reply:
Done